google-cloud-logging/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/cloud/logging/Logging</className>
+    <method>* deleteLog*(*)</method>
```

Comment:
nit: please, narrow the declaration at least the arguments. It will emphasis which methods were actually added since the last major version upgrade.